### ISSUE TITLE
`highlightStrokeProperty` has insufficient contrast with sim background

### ISSUE THREAD

**Issue by arouinfar** [phetsims/circuit-construction-kit-common#1228]
For #1178 

Yellow has insufficient color contrast with the sim background. (For light rays, this is an essential exception.)

<img width="328" height="365" alt="Image" src="https://github.com/user-attachments/assets/c3886a93-ee97-4008-94bd-4e5ccb0e3a78" />

A natural choice would be to use black instead. Black also has sufficient contrast with the red outlined used for proposed connections. Here's a rough mockup:

<img width="558" height="500" alt="Image" src="https://github.com/user-attachments/assets/d4a685e5-2b99-4bb6-ae13-b66aba83e29c" />

I'm not sure if I'm just so used to seeing yellow, but I don't love the black. However, it worked out for the knot highlight in FAMB in https://github.com/phetsims/forces-and-motion-basics/issues/404, so perhaps I'll like it better once I see it in action.

@samreid can you please update `highlightStrokeProperty` to black?

**Comment by samreid** [comment]
I updated it, and I'm having the same trouble of not knowing if I'm just used to the yellow, or if it was actually better. @arouinfar can you please test?
